qemu/tests/hugepage_mem_stress.py

```diff
@@ -29,40 +35,55 @@ def heavyload_install():
             install_cmd = params["install_cmd"] % winutil_drive
             session.cmd(install_cmd)
 
+    if params.get("non_existent_point"):
+        error_context.context("This path %s, doesn't mount hugepage." %
+                              params.get("non_existent_point"), logging.info)
+        if os.path.ismount("%s" % params.get("non_existent_point")):
+            process.run("umount %s" % params.get("non_existent_point"))
```

Comment:
Please use `tempfile.mkdtemp` to prepare the directory, here is the reference.

https://docs.python.org/3.6/library/tempfile.html?highlight=mkdtemp#tempfile.mkdtemp

Reply:
ok, thanks for the reminder.
correcting